master/buildbot/master.py

```diff
@@ -132,6 +133,9 @@ def create_child_services(self):
         self.metrics = metrics.MetricLogObserver()
         self.metrics.setServiceParent(self)
 
+        self.stats_service = stats_service.StatsService(self)
```

Comment:
ok if we can use the c['services'] registration point. At the end it is just another service. Storage services are actually subservices, there maybe some thing to tune here (using setParent, etc)?